CodeOfConduct.md

```diff
@@ -0,0 +1,133 @@
+or by filling out
+[this form](https://docs.google.com/forms/d/e/1FAIpQLSd8Tbi2zNl1i2N9COX0yavHEqTGFIPQ1_cLcy1A3JgVc1OrAQ/viewform).
+
+This code applies equally to founders, developers, mentors and new community
+members, in all spaces managed by the GeoPandas Project. This includes the
+mailing lists, our GitHub organization, our chat room, in-person events, and any
+other forums created by the project team. In addition, violations of this code
+outside these spaces may affect a person's ability to participate within them.
+
+By embracing the following principles, guidelines and actions to follow or
+avoid, you will help us make the GeoPandas Project a welcoming and productive
+community. Feel free to contact the Code of Conduct Committee at
+[geopandas-conduct@googlegroups.com](mailto:geopandas-conduct@googlegroups.com)
+with any questions.
+
+1. **Be friendly and patient**.
+
+2. **Be welcoming**. We strive to be a community that welcomes and supports
+   people of all backgrounds and identities. This includes, but is not limited
+   to, members of any race, ethnicity, culture, national origin, color,
+   immigration status, social and economic class, educational level, sex, sexual
+   orientation, gender identity and expression, age, physical appearance, family
+   status, technological or professional choices, academic
+   discipline, religion, mental ability, and physical ability.
+
+3. **Be considerate**. Your work will be used by other people, and you in turn
+   will depend on the work of others. Any decision you take will affect users
+   and colleagues, and you should take those consequences into account when
+   making decisions. Remember that we're a world-wide community. You may be
+   communicating with someone with a different primary language or cultural
+   background.
+
+4. **Be respectful**. Not all of us will agree all the time, but disagreement is
+   no excuse for poor behavior or poor manners. We might all experience some
+   frustration now and then, but we cannot allow that frustration to turn into a
+   personal attack. It’s important to remember that a community where people
+   feel uncomfortable or threatened is not a productive one.
+
+5. **Be careful in the words that you choose**. Be kind to others. Do not insult
+   or put down other community members. Harassment and other exclusionary
+   behavior are not acceptable. This includes, but is not limited to:
+
+    - Violent threats or violent language directed against another person
+    - Discriminatory jokes and language
+    - Posting sexually explicit or violent material
+    - Posting (or threatening to post) other people's personally identifying
+      information ("doxing")
+    - Personal insults, especially those using racist, sexist, and xenophobic terms
+    - Unwelcome sexual attention
+    - Advocating for, or encouraging, any of the above behavior
+    - Repeated harassment of others. In general, if someone asks you to stop,
```

Comment:
@brendan-ward I wouldn't exactly say duplicative, but instead circular. The way I read it is essentially "Harassment and other exclusionary behavior are not acceptable. This includes [...] repeated harassment of others." Or, "Harassment includes repeated harassment." So I can't quite tell how that bullet point helps clarify the definition of "harassment and other exclusionary behavior". To me, it would make more sense to excise that bullet point entirely, and instead conclude the section with the outside-the-bullet-points statement "In general, if someone asks you to stop, then stop."